objc2/objc.go

```diff
@@ -0,0 +1,52 @@
+package objc2
```

Comment:
I started a new package for this to experiment with the minimal amount of code needed to support the generated definitions. Based on how this goes we could either integrate the dynamic `Send` methods directly, or move that into a separate package that could be called on an object `Send(obj, "selector:")`